Pollo/ViewControllers/PollBuilderViewController+Extension.swift

```diff
@@ -61,6 +61,22 @@ extension PollBuilderViewController: MCPollBuilderViewDelegate {
         mcPollBuilder.fillDraft(title: draft.text, options: draft.options)
         loadedMCDraft = draft
         updateCanDraft(true)
+        for d in drafts {
+            if d.id == draft.id {
+                d.isLoaded = draft.isLoaded
+            }
+            else {
+                d.isLoaded = false
+            }
+        }
```

Comment:
you can make this more swifty and simple by doing:
```
drafts.forEach { $0.isLoaded = $0.id == draft.id ? draft.isLoaded : false }
```

Reply:
very nice